# test Message
# Do not use, testing only
Header header

string[] name
uint32[] index
float64[] damping
float64[] kp_position
float64[] ki_position
float64[] kd_position
float64[] kp_velocity
float64[] i_effort_min
float64[] i_effort_max
